m4/autoconf-archive/ax_check_compile_flag.m4

```diff
@@ -0,0 +1,72 @@
+# ===========================================================================
+#   http://www.gnu.org/software/autoconf-archive/ax_check_compile_flag.html
+# ===========================================================================
```

Comment:
Then I'd say that ac-archive is not available in Linaro CI setup, so let's bring the file in. Current message is just misleading.